pkg/sdk/config.go

```diff
@@ -12,8 +12,8 @@ import (
 func DefaultConfig() *gosnowflake.Config {
 	config, err := ProfileConfig("default")
 	if err != nil || config == nil {
-		log.Printf("[DEBUG] No Snowflake config file found, falling back to environment variables: %v\n", err)
-		return EnvConfig()
+		log.Printf("[DEBUG] No Snowflake config file found, returning empty config: %v\n", err)
```

Comment:
i feel like we should be able to read from environment variables, but maybe to avoid conflict, we can slightly rename them? LIke SF_USER instead of SNOWFLAKE_USER?

Reply:
I agree. But as I stated in the PR description: `removed fallback to environment variables for the SDK client (we will address the setup of the client separately with the extraction of the SDK client from this repository)`. This will matter after we extract the SDK into its own being. Before that, we are trying env vars first, so there is no need to do that here.